webrtc-stats.html

```diff
@@ -2049,6 +2050,20 @@ <h2>
                   MediaStreamTrack, if <code>type</code> is <code>"track"</code>).
                 </p>
               </dd>
+              <dt>
+                <dfn><code>hugeFramesSent</code></dfn> of type <span class="idlMemberType">
+                <a>unsigned long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the total number of huge frames sent by this RTCRtpSender (or for this
+                  MediaStreamTrack, if <code>type</code> is <code>"track"</code>). Huge frames, by
+                  definition, are frames with encoded size at least 2.5 times the target bitrate
+                  per second divided by target fps set for the encoder at the time. These are
```

Comment:
targetBitrate is a property of RTCOutboundRTPStreamStats, while this is a property of the RTCVideoHandlerStats.
framesPerSecond is a property of RTCVideoHandlerStats, so that's easily resolvable, but targetBitrate has to be specified as "the targetBitrate of the RTP stream that is currently sending this track" or something like that. They're connected via the trackId variable of the RTCRTPStreamStats.

Reply:
I don't see the issue here. The exact frame encoded size is not the part of any stats struct, yet it's needed for calculation of hugeFramesSent. So, tying this exact stat to something not present in RTCVideoHandlerStats is not an issue. All that information is  available inside the WebRTC where it computes the new stat.

It's already stated there, that bitrate and fps are that used by the encoder, which produced the encoded frame for that frame.

To reduce confusion, I will shortly rephrase it to: "... target bitrate per second divided by target fps set for the encoder at the time the frame was encoded"